README.md

```diff
@@ -17,27 +17,27 @@ __IMPORTANT NOTICE:__ The contents of this repository currectly reflect a __DRAF
-   1. [EiffelSourceChangeSubmittedEvent](./eiffel-vocabulary/EiffelSourceChangeSubmittedEvent.md)
-   1. [EiffelFlowContextDefinedEvent](./eiffel-vocabulary/EiffelFlowContextDefinedEvent.md)
-   1. [EiffelTestCaseStartedEvent](./eiffel-vocabulary/EiffelTestCaseStartedEvent.md)
-   1. [EiffelTestCaseFinishedEvent](./eiffel-vocabulary/EiffelTestCaseFinishedEvent.md)
-   1. [EiffelTestSuiteStartedEvent](./eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)
-   1. [EiffelTestSuiteFinishedEvent](./eiffel-vocabulary/EiffelTestSuiteFinishedEvent.md)
-   1. [EiffelIssueVerifiedEvent](./eiffel-vocabulary/EiffelIssueVerifiedEvent.md)
-   1. [EiffelTestExecutionRecipeCollectionCreatedEvent](./eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md)
-   1. [EiffelAnnouncementPublishedEvent](./eiffel-vocabulary/EiffelAnnouncementPublishedEvent.md)
-   1. [EiffelConfigurationAppliedEvent](./eiffel-vocabulary/EiffelConfigurationAppliedEvent.md)
+   1. [EiffelActivityTriggeredEvent (ActT)](./eiffel-vocabulary/EiffelActivityTriggeredEvent.md)
+   1. [EiffelActivityCanceledEvent (ActC)](./eiffel-vocabulary/EiffelActivityCanceledEvent.md)
+   1. [EiffelActivityStartedEvent (ActS)](./eiffel-vocabulary/EiffelActivityStartedEvent.md)
+   1. [EiffelActivityFinishedEvent (ActF)](./eiffel-vocabulary/EiffelActivityFinishedEvent.md)
+   1. [EiffelArtifactCreatedEvent (ArtC)](./eiffel-vocabulary/EiffelArtifactCreatedEvent.md)
+   1. [EiffelArtifactPublishedEvent (ArtP)](./eiffel-vocabulary/EiffelArtifactPublishedEvent.md)
+   1. [EiffelArtifactReusedEvent (ArtR)](./eiffel-vocabulary/EiffelArtifactReusedEvent.md)
+   1. [EiffelConfidenceLevelModifiedEvent (CLM)](./eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md)
+   1. [EiffelEnvironmentDefinedEvent (ED)](./eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md)
+   1. [EiffelCompositionDefinedEvent (CD)](./eiffel-vocabulary/EiffelCompositionDefinedEvent.md)
+   1. [EiffelSourceChangeCreatedEvent (SCC)](./eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md)
+   1. [EiffelSourceChangeSubmittedEvent (SCS)](./eiffel-vocabulary/EiffelSourceChangeSubmittedEvent.md)
+   1. [EiffelFlowContextDefinedEvent (FCD)](./eiffel-vocabulary/EiffelFlowContextDefinedEvent.md)
+   1. [EiffelTestCaseStartedEvent (TCS)](./eiffel-vocabulary/EiffelTestCaseStartedEvent.md)
+   1. [EiffelTestCaseFinishedEvent (TCF)](./eiffel-vocabulary/EiffelTestCaseFinishedEvent.md)
+   1. [EiffelTestSuiteStartedEvent (TSS)](./eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)
+   1. [EiffelTestSuiteFinishedEvent (TSF)](./eiffel-vocabulary/EiffelTestSuiteFinishedEvent.md)
+   1. [EiffelIssueVerifiedEvent (IV)](./eiffel-vocabulary/EiffelIssueVerifiedEvent.md)
+   1. [EiffelTestExecutionRecipeCollectionCreatedEvent (TERCC)](./eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md)
+   1. [EiffelAnnouncementPublishedEvent (AP)](./eiffel-vocabulary/EiffelAnnouncementPublishedEvent.md)
```

Comment:
I assume that using Art and Act is to prevent collisions and to give all a unique acronym. The AP is unique but it could easily be mistaken for a ArtP unless you are familiar with the acronyms. Should we rename to AnnP?